### ISSUE TITLE
File type icon not accessible in files app

### ISSUE THREAD

**Issue by Bubballoo3** [OSC/ondemand#4889]
See https://github.com/OSC/ondemand/pull/4877#discussion_r2632658639 for more details, but the file/folder icons we show in the first column have a title attribute, but this is not used by screenreaders (at least not reliably). The reliable fix I found for the project directory table was to add a `span.sr-only` with the title text.
